src/query/query_asset_scope_attribute.rs

```diff
@@ -158,72 +168,33 @@ mod tests {
             },
         },
         testutil::{
-            test_constants::{
-                DEFAULT_ASSET_TYPE, DEFAULT_ASSET_UUID, DEFAULT_SCOPE_SPEC_ADDRESS,
-                DEFAULT_SENDER_ADDRESS, DEFAULT_VERIFIER_ADDRESS,
-            },
-            test_utilities::{
-                mock_scope, mock_scope_attribute, test_instantiate_success, InstArgs,
-            },
+            test_constants::{DEFAULT_ASSET_UUID, DEFAULT_SCOPE_SPEC_ADDRESS},
+            test_utilities::{mock_scope, test_instantiate_success, InstArgs},
         },
-        util::scope_address_utils::asset_uuid_to_scope_address,
     };
 
     use super::query_asset_scope_attribute;
 
     #[test]
     fn test_successful_query_result() {
         let mut deps = mock_dependencies(&[]);
-        test_instantiate_success(deps.as_mut(), InstArgs::default());
-        let asset_uuid = "0caf9164-9f16-11ec-9a49-2b2175f69a81".to_string();
-        let scope_address = asset_uuid_to_scope_address(&asset_uuid)
-            .expect("expected uuid to scope address conversion to work properly");
-        // TDOO: Use the onboard_asset code here when it's ready with mock attribute tracking and such
```

Comment:
This was a TODO I missed from a long time ago that I decided to patch up.  I missed it because it had a typo lol and my search didn't pull it up

Reply:
TDOO!!